TechAndCheck/zenodotus
# Add video search

Add video search

This adds full reverse video search with the same interface as we
already had for images. From the user's standpoint it's the same, but
the backend is all sorts of new things.

This utilizes a new gem I created: [zelkova](https://github.com/cguess/zelkova) Which builds the search tree. Feel free to include the gem in this review if you'd like, or just for reference.

To test:
rails db:migrate
rails db:reset
rails t

start the server, click "media search" and then upload a video to see if
anything shows up. You may consider archiving a youtube video, and then
searching using the same video file to make sure it returns stuff.
